processor.go

```diff
@@ -47,6 +50,11 @@ func (p *Processor) Process(s interface{}) error {
 
 	mark(root, "")
 
+	if _, ok, _ := Args.Retrieve("help"); ok {
```

Comment:
Is there any way for you to use a constant +  comment for this ?

or better provide a way for user to choose the `--thing` they want to use. Using `--help` might conflict with --help command that could be present in existing software that could include the library.